anta/cli/get/commands.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+# coding: utf-8 -*-
+# pylint: disable = redefined-outer-name
```

Comment:
Can we get rid of this pylint ignore?

Reply:
Not easily and making code more complex:

It is related because we have an option named `tags` in `anta get inventory` and we also have a function named `tags` for `anta get tags`